### PR TITLE
Do not source from actors

### PR DESCRIPTION
**Pull request by jflatow**
This can lead to multiple constraints violating each other, which is causing scen failures. Not sure when/why we needed this but we should find another way if sourcing without it is not enough.

### CODE REVIEW

File: scenario/context/CometContext.ts [hunk 1]
```diff
@@ -210,7 +210,7 @@ export class CometContext {
     let amountRemaining = BigInt(amount);
 
     // Try to steal from a known whale or actor
-    for (const [name, actor] of whales.concat(Object.entries(this.actors))) {
+    for (const [name, actor] of whales) {
```
Comment (scott-silver) [inline scenario/context/CometContext.ts:213]: nit: update comment: `// Try to steal from a known whale`

**Review by scott-silver — APPROVED** [review]

File: scenario/context/CometContext.ts (hunk 1 above)
Comment (jflatow) [reply scenario/context/CometContext.ts:213]: Yeah, actually cleaning this up even further now

File: scenario/LiquidationBotScenario.ts [hunk 2]
```diff
@@ -443,7 +443,7 @@ scenario(
     },
     cometBalances: {
       albert: {
-        $asset0: ' == 5',
+        $asset0: ' == 10',
```
Comment (jflatow) [inline scenario/LiquidationBotScenario.ts:446]: Due to prices falling, we were apparently not borrowing enough anymore (below min)

File: scenario/TransferScenario.ts [hunk 3]
```diff
@@ -92,8 +92,8 @@ scenario(
     const baseTransferred = getExpectedBaseBalance(50n * scale, baseIndexScale, baseSupplyIndex);
     const baseOfTransferrer = getExpectedBaseBalance(baseSupplied - toTransfer, baseIndexScale, baseSupplyIndex);
 
-    expect(await comet.balanceOf(albert.address)).to.be.equal(baseOfTransferrer);
-    expect(await comet.balanceOf(betty.address)).to.be.equal(baseTransferred);
+    expectBase(await albert.getCometBaseBalance(), baseOfTransferrer);
+    expectBase(await betty.getCometBaseBalance(), baseTransferred);
```
Comment (jflatow) [inline scenario/TransferScenario.ts:96]: Unrelated but intermittently failing while testing this

Comment (scott-silver) [reply scenario/TransferScenario.ts:96]: `comet.balanceOf(actor)` and `actor.getCometBaseBalance()` aren't identical. looks like `getCometBaseBalance` subtracts the borrow balance.

was the original test incorrect?

Comment (scott-silver) [reply scenario/TransferScenario.ts:96]: oh, I see `expectBase` does an approximate match, not an exact one

**Comment by scott-silver** [comment]
re-approving. LGTM ✅

**Pull request merged**